evm/gas-service.js

```diff
@@ -45,6 +45,22 @@ function printFailedChainUpdates() {
     }
 }
 
+const feesCache = {};
+
+async function getFeeData(api, chain) {
+    if (!feesCache[chain]) {
+        const data = await httpPost(`${api}/gmp/getFees`, {
+            sourceChain: 'ethereum',
```

Comment:
You're assuming that `ethereum` is the valid name of the chain on that network. I'd suggest just picking the first chain name from the config as the one to use as the reference.